crates/parcel_filesystem/src/in_memory_file_system.rs

```diff
@@ -22,39 +22,26 @@ pub struct InMemoryFileSystem {
 
 impl InMemoryFileSystem {
   /// Change the current working directory. Used for resolving relative paths.
-  pub fn set_current_working_directory(&self, cwd: PathBuf) {
+  pub fn set_current_working_directory(&self, cwd: &Path) {
+    let cwd = self.canonicalize_impl(cwd);
```

Comment:
Can we fix the trait in these changes? I'm not a fan of the `_base` fn, we should just have `canonicalize` and the one that needs a cache should be named explicitly... something like `canonicalize_with_cache`

Reply:
There're a few things I think we could change in `trait FileSystem`, but that's really not the intention of this PR.

I just wanted a non-failable version of canonicalize for use internally.

Reply:
Yeah, I understand that. A follow up at some point would be nice.